openreview/conference/helpers.py

```diff
@@ -806,7 +812,15 @@ def get_submission_revision_stage(request_forum):
     if request_forum.content.get('accepted_submissions_only', '') == 'Enable revision for accepted submissions only':
         only_accepted = True
 
-    allow_author_reorder = request_forum.content.get('submission_author_edition', '') == 'Allow reorder of existing authors only'
+    author_reorder_map = {
+        'Allow reorder of existing authors only': openreview.stages.AuthorReorder.ALLOW_REORDER,
+        'Allow addition and removal of authors': openreview.stages.AuthorReorder.ALLOW_EDIT,
+        'Do not allow any changes to author lists': openreview.stages.AuthorReorder.DISALLOW_EDIT
+    }
+    allow_author_reorder = author_reorder_map[request_forum.content.get('submission_author_edition', 'Allow addition and removal of authors')]
+
+    if request_forum.content.get('api_version', '1') == '1':
```

Comment:
I am adding a third option to the request form for author reorder, but ignoring it if venue is API1

Reply:
sure, let's stop giving support to API 1 venues.